src/proxy/momento/src/protocol/redis/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod get;
```

Comment:
Is it intentional to name this module `redis` instead of `resp`?

Reply:
Fixed